docs/ops/deployment/kubernetes.md

```diff
@@ -319,6 +341,8 @@ spec:
         ports:
         - containerPort: 6122
           name: rpc
+        - containerPort: 6125
```

Comment:
Btw the user actually also needs a `Service` for the `query-state` port or something else, like we documented for REST. Maybe, this could be a follow-up improvement if not in this PR.

Reply:
I will add one more commit for your comments since it is a new added service.